python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -48,26 +48,29 @@ cpdef generate_pandas_metadata(Table table, index):
     # Indexes
     if index is not False:
         for name in table._index.names:
-            if isinstance(table._index, cudf.core.multiindex.MultiIndex):
-                idx = table.index.get_level_values(name)
+            if name is not None:
```

Comment:
The code to reproduce #4161 indicates the dataframe is produced and controlled by cudf. If there a value reason for `name` to be `None`?

Reply:
I can't speak for sure if its "valid" for `name` to be `None` but it certainly is `None` when no index exists, other than the default index.

Reply:
@kkraus14 Should we be getting a `None` here? If so, the conditional on `name is not None` works, but if not maybe we should find out what's happening that gets us to this point?

Reply:
Pandas has `None` as an index name, it looks like (`print(df.index.names[0])`).